plugins/usi-dock/fu-usi-dock-mcu-device.c

```diff
@@ -718,12 +735,15 @@ fu_usi_dock_mcu_device_cleanup(FuDevice *device,
 	fwupd_request_add_flag(request, FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE);
```

Comment:
this has the wrong id -- you need to replace `FWUPD_REQUEST_ID_INSERT_USB_CABLE` with `FWUPD_REQUEST_ID_REMOVE_USB_CABLE`